### PR TITLE
feat : added Card Component

### PR DESCRIPTION
**Pull request by omkarpatke**

### CODE REVIEW

**Comment by omkarpatke** [comment]
added Card with Badge,
Card with dismiss,
Card with Overlay,
Text Only Card,
Horizontal Card,
Shadow Card.

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.card-content{
+    padding: 0.5rem;
+}
+
+.text-badge{
+    position: relative;
+    top: 2rem;
+    background-color: #E67A00;
```
Comment (nikhiljugale007) [inline components/card.css]: You can use CSS variable here for background-color

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    color: var(--white-color);
+    background-color: var(--primary-color);
+    font-size: 1rem;
+    border: none;
+    border-radius: 10px;
+}
+
+.add-to-cart:hover{
+    background-color: var(--dark-primary-color);
+}
```
Comment (Naman-Saxena1) [inline components/card.css:89]: I am not sure if this is really getting used.

**Comment by nikhiljugale007** [comment]
You can change cursor to pointer on button hover and also can you a more darker color on hover so that it can be more distinguishable
Other than that UI and code looks good.

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    margin-left: 0.5rem;
+    opacity: .5;
+}
+
+.prices{
+    display: flex;
+    gap: 7px;
+    margin-top: .5rem;
+    margin-bottom: .5rem;
+}
```
Comment (Naman-Saxena1) [inline components/card.css:52]: For less than 1 rem, Vikram said we should use px.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
You can align your components to the center for symmetry and consistency.

File: components/card.css (hunk 1 above)
Comment (omkarpatke) [reply components/card.css:31]: thanks, Nikhil adding the variable for bg color.

File: components/card.css (hunk 2 above)
Comment (omkarpatke) [reply components/card.css:89]: sorry problem is in the dark primary color.

File: components/card.css (hunk 3 above)
Comment (omkarpatke) [reply components/card.css:52]: thanks for giving clarification, I'm really confused about that.

**Comment by nikhiljugale007** [comment]
Good to merge.

**Pull request merged**